jhaagsma/ee_npc
# Better selling of goods, add stockpiling

### REQUIRED
- remaining TODOs in code

### REQUIRED (TESTING)
- techers aim for 60 bpt
- mil reselling and buying more based on market conditions
- farmers shouldn't sell bushels at $100 or above under normal conditions ($50 early set)
- farmers are a bit less cs heavy at start, sell food on PM during protection
- on solo servers, bots reject allies and don't send them
- countries should stockpile
- add cashing too early country error
- https://github.com/jhaagsma/ee_npc/issues/20
- techers grow in a more reasonable fashion
- demos can bushel recycle during the set under certain conditions
- make recall goods function
- make recall tech function
- destock: techers can sell tech at reasonable prices and recall when it's time to dump tech at server min (only dump on Express)
- destock: let techers tech other things than mil tech once mil tech reaches goal
- destock: farmers can sell food at demo recycle max - 1 (recall stuck food at end)
- destock: all strats can recall bushels at very end	
- better tech production percentages
- better indy production percentages
- farmers should sell food on PM more often 
- farmers shouldn't cash instead of selling on private
- better farmer food sell prices - but it should usually be possible to sell above demo recycle limit (especially late in set)
- better tech sell prices
- better indy sell prices

### BUMPED

- can I get farmers selling food on public within 180 turns? would need to slow down cs I think, but that's probably a good thing
- https://github.com/jhaagsma/ee_npc/issues/21
- if high indy stored turns, recall goods (sell military on PM if needed), then dump goods at cheap prices on public
- get logout hours, then bump up max sell time from 8 hours if it would help? (Express only)
- stop indies from committing NW suicide if they get too many turns - can go into a cycle of continually selling 10% of military on PM	
- save exit condition to pref
